### DISCUSSION TITLE
v0.2.0

### DISCUSSION THREAD

**Discussion by cjbarrie** [cjbarrie/academictwitteR#178] [General]
This is a discussion board for the latest CRAN Release of academictwitteR v0.2.0

**Comment by shmuhammadd** [comment]
Hi @cjbarrie thanks for the good package. 

Please, when getting tweets from a user, I understand that we can use `user` and supplied the user handles we want get the data from.

What about extracting replies to a particular user, e.g all replies from `jack's` tweets. 

The sample code below does not work:



```
get_all_tweets(
  query = c("good"),
  user =  c("to:jack"),
  start_tweets = "2021-01-01T00:00:00Z",
  end_tweets = "2021-05-01T00:00:00Z",
  n = 30
)
```

Please, kindly advise. 

Thanks for the effort.
shamsuddeen

**Reply by cjbarrie** [reply]
Hi, thanks for using!

Try:

```r
get_all_tweets(
  query = c("good"),
  reply_to =  c("jack", "cbarrie"),
  start_tweets = "2021-01-01T00:00:00Z",
  end_tweets = "2021-05-01T00:00:00Z",
  n = 30
)
```

**Reply by shmuhammadd** [reply]
> Hi, thanks for using!
> 
> Try:
> 
> ```r
> get_all_tweets(
>   query = c("good"),
>   reply_to =  c("jack", "cbarrie"),
>   start_tweets = "2021-01-01T00:00:00Z",
>   end_tweets = "2021-05-01T00:00:00Z",
>   n = 30
> )
> ```

This works. Thanks @cjbarrie
